src/navigation/MainNavigator.tsx

```diff
@@ -39,6 +39,7 @@ const headerScreenOptions: StackNavigationOptions = {
   },
   headerBackTitleVisible: false,
   headerTintColor: Colors.headerText,
+  headerTitleAlign: "center",
```

Comment:
Will this make the setting on nested navigators obsolete?

Reply:
I believe it serves as the default for any directly nested screens.